charts/stable/qbittorrent/values.yaml

```diff
@@ -36,30 +36,11 @@ service:
         port: 8000
         targetPort: 8000
         protocol: http
-workload:
-  qbitportforward:
-    enabled: true
-    type: Deployment
-    strategy: RollingUpdate
-    replicas: 1
-    podSpec:
-      containers:
-        qbitportforward:
-          primary: true
-          enabled: true
-          imageSelector: qbitportforwardImage
-          probes:
-            liveness:
-              enabled: false
-            readiness:
-              enabled: false
-            startup:
-              enabled: false
-          env:
-            QBT_USERNAME: "{{ .Values.qbitportforward.QBT_USERNAME }}"
-            QBT_PASSWORD: "{{ .Values.qbitportforward.QBT_PASSWORD }}"
-            QBT_ADDR: '{{ printf "http://%v:%v" (include "tc.v1.common.lib.chart.names.fullname" $) .Values.service.main.ports.main.port }}'
-            GTN_ADDR: '{{ printf "http://%v-gluetun:8000" (include "tc.v1.common.lib.chart.names.fullname" $) }}'
+cronjob:
```

Comment:
The should go too, use .Values.qbitportforward.{cron values,etc}

Reply:
Do I still need this?